roles/ceph-common/defaults/main.yml

```diff
@@ -7,7 +7,7 @@ ceph:
   #  - sdc
   #  - sdd
 
-  ceph_version: v0.94.3
+  ceph_version: v0.94.5
```

Comment:
This seems unrelated to the change. Shoudl be it's own change or accounted for in the commit message / summary. Probably best if it's own commit.